tools/pylint-extensions/pylint-guidelines-checker/pylint_guidelines_checker.py

```diff
@@ -1068,56 +1087,6 @@ def visit_classdef(self, node):
             pass
 
 
-class PackageNameDoesNotUseUnderscoreOrPeriod(BaseChecker):
```

Comment:
Did this get removed? If so, why?

Reply:
Our pylintrc file has us skipping setup.py for pylint checks, so this rule never actually is used here. I made an [issue](https://github.com/Azure/azure-sdk-for-python/issues/28202)to investigate if this is a check we want in another part of the pipeline instead.